### ISSUE TITLE
Lost ability to change 'Library' location using Preferences following upgrade to 3.7.4 

### ISSUE THREAD

**Issue by ajwhiteley** [openaudible/openaudible#1197]
I've just upgraded to 3.7.4 on an older iMac running Mac Catalina 10.15.7. 

TL; DR
1.  I'd like to be able set/configure the 'Library' location as I have been able to do in previous versions (this has gone missing and using environment variables seems a bit weird for such a major feature). There's a screenshot below. 
2. I'd like to store media (and converted media) separately from the config,  metadata. 
3. I'd like to save converted m4b, mp3s once rather than twice as I have been doing (once in 'books' and again in format-specific 'web' directories).

In previous versions of OpenAudible, I have used Preferences to set/change the 'Library', 'Web Directory'  and 'Preferences' directories.  For some reason, I currently only have options to edit the 'Web Directory' and 'Preferences'; the ability to change the  'Library' directory has entirely disappeared following the update. Having set this using  OPENAUDIBLE_HOME environment variable didn't achieve what I was hoping either. Currently, OpenAudible can't find any of my previously downloaded, converted books, and converted books. 

I have previously changed the locations of the 'Web Directory' and 'Library' directories from their defaults to allow the storage of the Library on an external hard drive as I had issues storing directly on an nfs-mounted NAS (for the same reason mentioned in issues here somewhere). 

I have over 1,000 books. If I convert my books to both  mp3 and m4b and store them, along with the AAX files, I have run out of space on a 2tb drive. It would be nice to not have to keep copies in both 'Books' and the associated directories under 'web' for both types of converted files. 

I want all of my preferences, locations, configs, metadata etc. stored in my ~/Library/OpenAudible directory. I want to store pretty much EVERYTHING else (.AAX, mp3, m4p, html, images) somewhere else, e.g. on an external drive. I haven't messed with the contents of 'directories.json' file; it isn't obvious to me looking at the keys which one is where the AAX  m4b, mp3 are stored in the AAX and Books directories respectively. 

![OpenAudible-3 7 4-Prefs](https://user-images.githubusercontent.com/16036220/232324724-e1a19050-c87a-4366-9e93-20126cb3ede8.png)

This is what my storage looks like (external storage):
726G	./books (this is the m4b and mp3s together in the same directory currently) <<-- this is the duplication
16k  	./tmp
730G	./web (mp3s and m4bs are stored in separate directories under web) 
56M	        ./art
394G	./aax

I'd like to not have to store the AAX files anywhere in $HOME. I'm fine with storing the metadata, configs, etc. in $HOME. I want to store the converted versions of the native AAX files somewhere that the generated html understands (one of my favourite features). I'd like to be able to store converted files once, ideally in the web location. 

thanks in advance

**Comment by openaudible** [comment]
Under File> Manage Libraries

**Comment by ajwhiteley** [comment]
Found it. I had a horrible time trying to work out how to navigate to books.json under '~./Library'. Finder pretends not to know about 'Library' until it has been added to Favourites. Maybe there's another way. 

 I wasn't able to open an existing Library for some reason; operator error surely. So I had to create a new one and then move files into the new Library.
